tests/common.hpp

```diff
@@ -111,6 +111,15 @@ START_TEST(__testname) {                                                \
 #define BEGIN_CHECK do
 #define END_CHECK while(0)
 
+#define GDR_OPEN_SAFE() ({                                                              \
```

Comment:
I prefer inline functions over macros, unless strictly necessary to pass info regarding the original call site

Reply:
Thank you Davide. Fixed.